src/providers/BackgroundBlurProvider/index.tsx

```diff
@@ -33,29 +32,26 @@ interface Props extends BaseSdkProps {
   /** A set of options that can be supplied when creating a background blur video frame processor. For more information, refer to
    * [Amazon Chime SDK for JavaScript Background Filter Guide](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/15_Background_Filter_Video_Processor.md#adding-a-background-filter-to-your-application). */
   options?: BackgroundBlurOptions;
-  /**
-   * Observer callback functions. The observer will be added to the background blur processor on mount and removed on unmount.
-   */
-  observer?: BackgroundBlurVideoFrameProcessorObserver;
 }
 
 interface BackgroundBlurProviderState {
   createBackgroundBlurDevice: (
     device: Device
   ) => Promise<DefaultVideoTransformDevice>;
   isBackgroundBlurSupported: boolean | undefined;
+  backgroundBlurProcessor: BackgroundBlurProcessor | undefined
 }
 
 const BackgroundBlurProviderContext = createContext<
   BackgroundBlurProviderState | undefined
 >(undefined);
 
-const BackgroundBlurProvider: FC<Props> = ({ spec, options, observer, children }) => {
+const BackgroundBlurProvider: FC<Props> = ({ spec, options, children }) => {
   const logger = useLogger();
   const [isBackgroundBlurSupported, setIsBackgroundBlurSupported] = useState<
     boolean | undefined
   >(undefined);
-  const [processor, setProcessor] = useState<BackgroundBlurProcessor | undefined>();
+  const [backgroundBlurProcessor, setProcessor] = useState<BackgroundBlurProcessor | undefined>();
```

Comment:
It is generally a best practice/convention to have this naming as [state, setState]. Could you please change `setProcessor` to `setBackgroundBlurProcessor`?

Or keep the state name as `processor` and where you provide value:
```
{
  // other provider values...,
  backgroundBlurProcessor: processor
}
```

Reply:
Same for `BackgroundReplacementProviderState`.